test/unit/url-match-pattern-list.test.js

```diff
@@ -17,35 +17,39 @@ describe('UrlMatchPatternList', function () {
     describe('test', function () {
         specForTestMethodOnTarget(getTargetForSpec);
 
-        it('should match for any', function () {
+        it('should match for any', function (done) {
             var matchPatternList = new UrlMatchPatternList({}, ['https://example.com/*']);
-            expect(matchPatternList.test('https://example.com')).to.eql(true);
-            expect(matchPatternList.test('https://example.com/')).to.eql(true);
-            expect(matchPatternList.test('https://www.example.com/')).to.eql(false);
-            expect(matchPatternList.test('https://example.com/hello')).to.eql(true);
-            expect(matchPatternList.test('https://example.com/foo/bar')).to.eql(true);
-            expect(matchPatternList.test('https://foo.example.com')).to.eql(false);
-            expect(matchPatternList.test('https://foo.com')).to.eql(false);
+            expect(matchPatternList.test('https://example.com')).to.be.true;
+            expect(matchPatternList.test('https://example.com/')).to.be.true;
+            expect(matchPatternList.test('https://www.example.com/')).to.be.false;
+            expect(matchPatternList.test('https://example.com/hello')).to.be.true;
+            expect(matchPatternList.test('https://example.com/foo/bar')).to.be.true;
+            expect(matchPatternList.test('https://foo.example.com')).to.be.false;
+            expect(matchPatternList.test('https://foo.com')).to.be.false;
+            done();
```

Comment:
The `done` callback is not needed here or in any of the tests below where it has been added.